VirtoCommerce.WebhooksModule.Web/Scripts/blades/webhook-detail.js

```diff
@@ -44,6 +49,21 @@
         }
 
         function saveOrUpdate() {
+            var newEventsList = [];
+
+            angular.forEach(blade.currentEntity.events, function(selectedEvent) {
```

Comment:
Could you try to avoid such list synchronization? Using `blade.currentEntity.events` should be enough for the saving (maybe WebHookEvent.Patch changes would require).

Reply:
fixed